python/hail/__init__.py

```diff
@@ -56,11 +58,13 @@
            'agg',
            'genetics',
            'methods',
-           'linalg']
+           'linalg',
+           'plot']
 
 
 __all__.extend(genetics.__all__)
 __all__.extend(methods.__all__)
+__all__.extend(plot.__all__)
```

Comment:
delete. This line adds everything in the `plot` module to the top level `hl` namespace -- `hl.scatter`, `hl.histogram`, etc. We probably don't want this, at least not yet.